packages/code-studio/.env.development

```diff
@@ -1,5 +1,4 @@
 VITE_ENABLE_LOG_PROXY=false
-VITE_CORE_API_URL=http://localhost:10000/jsapi
```

Comment:
This is used in the `index.html` and should be updated/checked there. The JS API script tags link using this env variable

Reply:
It's loading the jsapi successfully, so I assume that means it works.

Reply:
> Should also update in `embed-grid` and `embed-chart` packages

They aren't setup with any proxy right now. I guess we could, but it's not necessary.